ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_linalg.py

```diff
@@ -763,6 +798,47 @@ def test_adjoint(dtype_x, test_flags, backend_fw, fn_name):
     )
 
 
+@handle_test(
+    fn_tree="functional.ivy.experimental.batched_outer",
+    data=_batched_outer_data(),
+)
+def test_batched_outer(*, data, test_flags, backend_fw, fn_name, on_device):
+    input_dtypes, tensors = data
+    if backend_fw == "paddle":
+        # to avoid large dimension results since paddle don't support them
+        tensors = tensors[:2]
```

Comment:
Don't follow this- could you please explain why this is needed?

Reply:
same reason as previously mentioned, more tensors result in bigger dimensions  in the result.